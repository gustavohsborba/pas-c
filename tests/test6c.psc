var
 a, b, c, maior is int;
 nome is string;
 start
 read(a);
 read(b);
 read(c;

 maior := 0;
 if ( a>b and a>c ) then
 maior := a;
 else
 if (b>c) then
 maior := b;
 else
 maior := c;
 end
 end
 Out({Maior idade: });
 out(maior);
 end