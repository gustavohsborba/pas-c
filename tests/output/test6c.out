Opening file tests/test6c.psc...
ERROR in line 5, column 6: Sintax error found - Expected TOK_IS,  but 'read' found
